woocommerce/woocommerce-android
# Add track event to see analytic report

Closes: #10668

### Why
We plan to add full analytics reports to the Analytics Hub, by opening the web report in a webview.  This PR tracks user interactions with the new See Report button on the Analytics Hub cards.

### Description
This PR triggers the new `analytics_hub_view_full_report_tapped` event when the user taps on a card’s link to view the full analytics report, with the custom props reflecting the selected report and the currently selected date range and comparison period.

Registered here: https://github.com/Automattic/tracks-events-registration/pull/2221.

#### Custom properties:
- Custom prop `report` with value `revenue`|`orders`|`products` (the report that is opened)
- Custom prop `period` with value set to the currently selected date range (e.g. `today`|`yesterday`|`custom`)
- Custom prop `compare` with value `previous_period` (in the future this can be expanded to include previous_year when the app supports a previous year comparison)

### Testing instructions
1. Open the My Store screen
2. Tap on See more
3. Tap on the See Report link of the different cards and check that the `analytics_hub_view_full_report_tapped` event is sent and the custom props reflect the selected report, the selected date range, and the comparison period.

### Images/gif

```
🔵 Tracked: analytics_hub_view_full_report_tapped, Properties: {"period":"today","report":"products","compare":"previous_period","blog_id":222408976,"is_wpcom_store":true,"was_ecommerce_trial":true,"plan_product_slug":"wooexpress-small-bundle-yearly","store_id":"b6468f83-4986-44f1-b4ce-6021649dbf29","is_debug":true,"site_url":"https:\/\/woo-practically-automatic-witch.wpcomstaging.com"}
```
